Jetstream/Client.swift

```diff
@@ -207,7 +207,7 @@ public typealias TransportAdapterFactory = () -> TransportAdapter
             if let this = self {
                 for (scope, params) in scopesAndFetchParams {
                     session.fetch(scope, params: params) { error in
-                        if let definiteError = error {
+                        if let _ = error {
                             this.logger.error("Received error refetching scope '\(scope.name)': \(error)")
```

Comment:
Perhaps logging out `definiteError` than `error` rather than declaring `_` as the variable in scope here might be better?  Otherwise it tends to print out `Optional(...)` when printing the value.